### ISSUE TITLE
problem with both auto install and manual install

### ISSUE THREAD

**Issue by tensor-programming** [fdopen/opam-repository-mingw#43]
When trying to install Opam and the Ocaml compiler via the manual, I receive these errors:

```
opam init mingw "https://github.com/fdopen/opam-repository-mingw.git" --comp 4                                     .05.0+mingw64c --switch 4.05.0+mingw64c
Checking for available remotes: rsync and local, git.
  - you won't be able to use mercurial repositories unless you install the hg
    command on your system.
  - you won't be able to use darcs repositories unless you install the darcs
    command on your system.

[WARNING] No external solver found, one of aspucd, packup and mccs is
          recommended (see
          http://opam.ocaml.org/doc/Install.html#ExternalSolvers for details)
[WARNING] Recommended dependencies -- most packages rely on these:
            - cc


=-=- Fetching repository information =-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
[mingw: git]
[mingw: git]
[mingw: git]
[mingw: git]
[mingw: git]
[mingw: git]
[mingw] git+https://github.com/fdopen/opam-repository-mingw.git updated

=-=- Installing compiler 4.05.0+mingw64c =-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
[4.05.0+mingw64c: http]
[compiler.get] https://dl.bintray.com/fdopen/prebuild-win32-ocaml-packages/4.05.                                     0+mingw64c2.tar.xz downloaded
Now compiling OCaml. This may take a while, please bear with us...
[4.05.0+mingw64c: sed]
[4.05.0+mingw64c: sed]
[4.05.0+mingw64c: dash.exe install.sh]
Done.

=-=- Gathering sources =-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=

=-=- Processing actions -=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=

#=== ERROR while installing base-unix.base ====================================#
OpamSystem.File_not_found("C:/Projects/.opam/4.05.0+mingw64c/lib/base-unix/opam.                                     config")

#=== ERROR while installing base-threads.base =================================#
OpamSystem.File_not_found("C:/Projects/.opam/4.05.0+mingw64c/lib/base-threads/op                                     am.config")

#=== ERROR while installing base-bigarray.base ================================#
OpamSystem.File_not_found("C:/Projects/.opam/4.05.0+mingw64c/lib/base-bigarray/o                                     pam.config")


=-=- Error report -=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
The following actions failed
  ∗  install base-bigarray base
  ∗  install base-threads  base
  ∗  install base-unix     base
No changes have been performed
[ERROR] Initialisation failed
OpamStd.OpamSys.Exit(4)
```

These three errors, `base-bigarray`, `base-threads` and `base-unix` consistently happen when using both the auto-install method and the manual install method as well as when using the prebuilt Ocaml compiler and the source. I've tried completely purging cygwin etc multiple times and keep encountering this error no matter what I try.  Is there something I am just blatantly missing here?  

Edit: this happens even with the 4.03 version and the older version mentioned in the wiki post.  I haven't tried the 32 bit version yet because I don't really think that would make a difference.  Also, c:\Projects is my default home (i set it a while ago for emacs).  I'm not sure this is the problem either.  Capital letters shouldn't cause an issue.

**Comment by fdopen** [comment]
Please post the output of

```bash
env
cmd /c set
mount
cygcheck -c
```

at the point before you start the manual installation (`opam init ...`).

**Comment by tensor-programming** [comment]
I removed all three Cygwin installations from the auto and manual installs.  Here is the output of a fresh cygwin install following the manual instructions. (i've replaced my username with xxxxx.)

env:
```
USERDOMAIN=DESKTOP-K8K84AK
OS=Windows_NT
COMMONPROGRAMFILES=C:\Program Files\Common Files
PROCESSOR_LEVEL=21
PSModulePath=C:\Program Files\WindowsPowerShell\Modules;C:\WINDOWS\system32\WindowsPowerShell\v1.0\Modules
CommonProgramW6432=C:\Program Files\Common Files
PKG_CONFIG=C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v8.0\include
MSBUILD_PATH=C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\MSBuild\15.0\Bin
HADOOP_HOME=C:\spark-2.2.0-bin-hadoop2.7\bin
CommonProgramFiles(x86)=C:\Program Files (x86)\Common Files
LANG=en_US.UTF-8
TZ=America/New_York
HOSTNAME=DESKTOP-K8K84AK
PUBLIC=C:\Users\Public
OLDPWD=/usr/bin
CUDA_PATH=C:\cuda\v8.0
USERNAME=XXXXX
JAVA_HOME=C:\Program Files\Java\jdk1.8.0_144
LOGONSERVER=\\DESKTOP-K8K84AK
PROCESSOR_ARCHITECTURE=AMD64
LOCALAPPDATA=C:\Users\XXXXX\AppData\Local
COMPUTERNAME=DESKTOP-K8K84AK
USER=XXXXX
!::=::\
SYSTEMDRIVE=C:
USERPROFILE=C:\Users\XXXXX
DART_SDK=C:\Program Files\Dart\dart-sdk
PATHEXT=.COM;.EXE;.BAT;.CMD;.VBS;.VBE;.JS;.JSE;.WSF;.WSH;.MSC;.RB;.RBW
SYSTEMROOT=C:\WINDOWS
GOPATH=C:\Projects\go\
CUDA_PATH_V8_0=C:\cuda\v8.0
USERDOMAIN_ROAMINGPROFILE=DESKTOP-K8K84AK
PROCESSOR_IDENTIFIER=AMD64 Family 21 Model 2 Stepping 0, AuthenticAMD
PWD=/cygdrive/c/Projects
STACK_ROOT=C:\sr
HOME=/cygdrive/c/Projects/
GOROOT=C:\Go\
FSHARPINSTALLDIR=C:\Program Files (x86)\Microsoft SDKs\F#\4.1\Framework\v4.0\
TMP=/tmp
OneDrive=C:\Users\XXXXX\OneDrive
!C:=C:\cygwin64\bin
PROCESSOR_REVISION=0200
SPARK_HOME=C:\spark-2.2.0-bin-hadoop2.7\
PROFILEREAD=true
PROMPT=$P$G
NUMBER_OF_PROCESSORS=8
ProgramW6432=C:\Program Files
COMSPEC=C:\WINDOWS\system32\cmd.exe
APPDATA=C:\Users\XXXXX\AppData\Roaming
SHELL=/bin/bash
TERM=cygwin
WINDIR=C:\WINDOWS
SC2PATH=C:\Program Files (x86)\StarCraft II
ProgramData=C:\ProgramData
SHLVL=1
PRINTER=HPAF47EF (HP Deskjet 3520 series)
PROGRAMFILES=C:\Program Files
ALLUSERSPROFILE=C:\ProgramData
TEMP=/tmp
GYP_MSVS_VERSION=2015
SESSIONNAME=Console
SBT_HOME=C:\Program Files (x86)\sbt\
ProgramFiles(x86)=C:\Program Files (x86)
PATH=/usr/local/bin:/usr/bin:/cygdrive/c/cuda/v8.0/bin:/cygdrive/c/cuda/v8.0/libnvvp:/cygdrive/c/ProgramData/Oracle/Java/javapath:/cygdrive/c/Windows/system32:/cygdrive/c/Windows:/cygdrive/c/Windows/System32/Wbem:/cygdrive/c/Windows/System32/WindowsPowerShell/v1.0:/cygdrive/c/Program Files (x86)/NVIDIA Corporation/PhysX/Common:/cygdrive/c/Program Files/nodejs:/cygdrive/c/Program Files/erl9.0/bin:/cygdrive/c/Users/XXXXX/.mix/escripts:/cygdrive/c/WINDOWS/system32:/cygdrive/c/WINDOWS:/cygdrive/c/WINDOWS/System32/Wbem:/cygdrive/c/WINDOWS/System32/WindowsPowerShell/v1.0:/cygdrive/c/Program Files/Git/cmd:/cygdrive/c/Program Files/Dart/dart-sdk/bin:/cygdrive/c/PostgreSQL/pg96/bin:/cygdrive/c/Program Files (x86)/Windows Kits/8.1/Windows Performance Toolkit:/cygdrive/c/Program Files/CMake/bin:/cygdrive/c/Program Files/Java/jdk1.8.0_144/bin:/cygdrive/c/Program Files (x86)/sbt/bin:/cygdrive/c/Go/bin:/cygdrive/c/Program Files (x86)/Elixir/bin:/cygdrive/c/Users/XXXXX/.mix/escripts:/cygdrive/c/Program Files/Git/mingw64/bin:/cygdrive/c/Program Files/dotnet:/cygdrive/c/Ruby24-x64/bin:/cygdrive/c/Users/XXXXX/AppData/Local/Programs/Python/Python36/Scripts:/cygdrive/c/Users/XXXXX/AppData/Local/Programs/Python/Python36:/cygdrive/c/Users/XXXXX/AppData/Roaming/local/bin:/cygdrive/c/Users/XXXXX/.cargo/bin:/cygdrive/c/Users/XXXXX/scoop/apps/yarn/current/Yarn/bin:/cygdrive/c/Users/XXXXX/scoop/apps/yarn/current/bin:/cygdrive/c/Users/XXXXX/scoop/shims:/cygdrive/c/Users/XXXXX/AppData/Local/Microsoft/WindowsApps:/cygdrive/c/emacs/bin:/cygdrive/c/flutter/bin:/cygdrive/c/Users/XXXXX/.cargo/bin:/cygdrive/c/Users/XXXXX/AppData/Local/atom/bin:/cygdrive/c/Users/XXXXX/AppData/Roaming/npm:/cygdrive/c/Program Files/Microsoft VS Code/bin:/cygdrive/c/boot-clj:/cygdrive/c/Users/XXXXX/AppData/Local/GitHubDesktop/bin:/cygdrive/c/apktool:/cygdrive/c/sciter/bin/64:/cygdrive/c/nw/nwjs-v0.25.2-win-x64:/cygdrive/c/spark-2.2.0-bin-hadoop2.7/bin:/cygdrive/c/Program Files (x86)/Microsoft Visual Studio 14.0/VC/bin/amd64:/cygdrive/c/Users/XXXXX/AppData/Roaming/Pub/Cache/bin:/cygdrive/c/Projects/go/bin:/cygdrive/c/lein:/cygdrive/c/Program Files (x86)/Microsoft SDKs/F#/4.1/Framework/v4.0
PS1=\[\e]0;\w\a\]\n\[\e[32m\]\u@\h \[\e[33m\]\w\[\e[0m\]\n\$
HOMEDRIVE=C:
INFOPATH=/usr/local/info:/usr/share/info:/usr/info
HOMEPATH=\Users\XXXXX
ORIGINAL_PATH=/cygdrive/c/cuda/v8.0/bin:/cygdrive/c/cuda/v8.0/libnvvp:/cygdrive/c/ProgramData/Oracle/Java/javapath:/cygdrive/c/Windows/system32:/cygdrive/c/Windows:/cygdrive/c/Windows/System32/Wbem:/cygdrive/c/Windows/System32/WindowsPowerShell/v1.0:/cygdrive/c/Program Files (x86)/NVIDIA Corporation/PhysX/Common:/cygdrive/c/Program Files/nodejs:/cygdrive/c/Program Files/erl9.0/bin:/cygdrive/c/Users/XXXXX/.mix/escripts:/cygdrive/c/WINDOWS/system32:/cygdrive/c/WINDOWS:/cygdrive/c/WINDOWS/System32/Wbem:/cygdrive/c/WINDOWS/System32/WindowsPowerShell/v1.0:/cygdrive/c/Program Files/Git/cmd:/cygdrive/c/Program Files/Dart/dart-sdk/bin:/cygdrive/c/PostgreSQL/pg96/bin:/cygdrive/c/Program Files (x86)/Windows Kits/8.1/Windows Performance Toolkit:/cygdrive/c/Program Files/CMake/bin:/cygdrive/c/Program Files/Java/jdk1.8.0_144/bin:/cygdrive/c/Program Files (x86)/sbt/bin:/cygdrive/c/Go/bin:/cygdrive/c/Program Files (x86)/Elixir/bin:/cygdrive/c/Users/XXXXX/.mix/escripts:/cygdrive/c/Program Files/Git/mingw64/bin:/cygdrive/c/Program Files/dotnet:/cygdrive/c/Ruby24-x64/bin:/cygdrive/c/Users/XXXXX/AppData/Local/Programs/Python/Python36/Scripts:/cygdrive/c/Users/XXXXX/AppData/Local/Programs/Python/Python36:/cygdrive/c/Users/XXXXX/AppData/Roaming/local/bin:/cygdrive/c/Users/XXXXX/.cargo/bin:/cygdrive/c/Users/XXXXX/scoop/apps/yarn/current/Yarn/bin:/cygdrive/c/Users/XXXXX/scoop/apps/yarn/current/bin:/cygdrive/c/Users/XXXXX/scoop/shims:/cygdrive/c/Users/XXXXX/AppData/Local/Microsoft/WindowsApps:/cygdrive/c/emacs/bin:/cygdrive/c/flutter/bin:/cygdrive/c/Users/XXXXX/.cargo/bin:/cygdrive/c/Users/XXXXX/AppData/Local/atom/bin:/cygdrive/c/Users/XXXXX/AppData/Roaming/npm:/cygdrive/c/Program Files/Microsoft VS Code/bin:/cygdrive/c/boot-clj:/cygdrive/c/Users/XXXXX/AppData/Local/GitHubDesktop/bin:/cygdrive/c/apktool:/cygdrive/c/sciter/bin/64:/cygdrive/c/nw/nwjs-v0.25.2-win-x64:/cygdrive/c/spark-2.2.0-bin-hadoop2.7/bin:/cygdrive/c/Program Files (x86)/Microsoft Visual Studio 14.0/VC/bin/amd64:/cygdrive/c/Users/XXXXX/AppData/Roaming/Pub/Cache/bin:/cygdrive/c/Projects/go/bin:/cygdrive/c/lein:/cygdrive/c/Program Files (x86)/Microsoft SDKs/F#/4.1/Framework/v4.0
EXECIGNORE=*.dll
VBOX_MSI_INSTALL_PATH=C:\Program Files\Oracle\VirtualBox\
VS140COMNTOOLS=C:\Program Files (x86)\Microsoft Visual Studio 14.0\Common7\Tools\
_=/usr/bin/env
```

cmd /c set:
```
ALLUSERSPROFILE=C:\ProgramData
APPDATA=C:\Users\XXXXX\AppData\Roaming
COMMONPROGRAMFILES=C:\Program Files\Common Files
COMPUTERNAME=DESKTOP-K8K84AK
COMSPEC=C:\WINDOWS\system32\cmd.exe
CUDA_PATH=C:\cuda\v8.0
CUDA_PATH_V8_0=C:\cuda\v8.0
CommonProgramFiles(x86)=C:\Program Files (x86)\Common Files
CommonProgramW6432=C:\Program Files\Common Files
DART_SDK=C:\Program Files\Dart\dart-sdk
EXECIGNORE=*.dll
FSHARPINSTALLDIR=C:\Program Files (x86)\Microsoft SDKs\F#\4.1\Framework\v4.0\
GOPATH=C:\Projects\go\
GOROOT=C:\Go\
GYP_MSVS_VERSION=2015
HADOOP_HOME=C:\spark-2.2.0-bin-hadoop2.7\bin
HOME=c:\Projects\
HOMEDRIVE=C:
HOMEPATH=\Users\XXXXX
HOSTNAME=DESKTOP-K8K84AK
INFOPATH=/usr/local/info:/usr/share/info:/usr/info
JAVA_HOME=C:\Program Files\Java\jdk1.8.0_144
LANG=en_US.UTF-8
LOCALAPPDATA=C:\Users\XXXXX\AppData\Local
LOGONSERVER=\\DESKTOP-K8K84AK
MSBUILD_PATH=C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\MSBuild\15.0\Bin
NUMBER_OF_PROCESSORS=8
OLDPWD=/usr/bin
ORIGINAL_PATH=/cygdrive/c/cuda/v8.0/bin:/cygdrive/c/cuda/v8.0/libnvvp:/cygdrive/c/ProgramData/Oracle/Java/javapath:/cygdrive/c/Windows/system32:/cygdrive/c/Windows:/cygdrive/c/Windows/System32/Wbem:/cygdrive/c/Windows/System32/WindowsPowerShell/v1.0:/cygdrive/c/Program Files (x86)/NVIDIA Corporation/PhysX/Common:/cygdrive/c/Program Files/nodejs:/cygdrive/c/Program Files/erl9.0/bin:/cygdrive/c/Users/XXXXX/.mix/escripts:/cygdrive/c/WINDOWS/system32:/cygdrive/c/WINDOWS:/cygdrive/c/WINDOWS/System32/Wbem:/cygdrive/c/WINDOWS/System32/WindowsPowerShell/v1.0:/cygdrive/c/Program Files/Git/cmd:/cygdrive/c/Program Files/Dart/dart-sdk/bin:/cygdrive/c/PostgreSQL/pg96/bin:/cygdrive/c/Program Files (x86)/Windows Kits/8.1/Windows Performance Toolkit:/cygdrive/c/Program Files/CMake/bin:/cygdrive/c/Program Files/Java/jdk1.8.0_144/bin:/cygdrive/c/Program Files (x86)/sbt/bin:/cygdrive/c/Go/bin:/cygdrive/c/Program Files (x86)/Elixir/bin:/cygdrive/c/Users/XXXXX/.mix/escripts:/cygdrive/c/Program Files/Git/mingw64/bin:/cygdrive/c/Program Files/dotnet:/cygdrive/c/Ruby24-x64/bin:/cygdrive/c/Users/XXXXX/AppData/Local/Programs/Python/Python36/Scripts:/cygdrive/c/Users/XXXXX/AppData/Local/Programs/Python/Python36:/cygdrive/c/Users/XXXXX/AppData/Roaming/local/bin:/cygdrive/c/Users/XXXXX/.cargo/bin:/cygdrive/c/Users/XXXXX/scoop/apps/yarn/current/Yarn/bin:/cygdrive/c/Users/XXXXX/scoop/apps/yarn/current/bin:/cygdrive/c/Users/XXXXX/scoop/shims:/cygdrive/c/Users/XXXXX/AppData/Local/Microsoft/WindowsApps:/cygdrive/c/emacs/bin:/cygdrive/c/flutter/bin:/cygdrive/c/Users/XXXXX/.cargo/bin:/cygdrive/c/Users/XXXXX/AppData/Local/atom/bin:/cygdrive/c/Users/XXXXX/AppData/Roaming/npm:/cygdrive/c/Program Files/Microsoft VS Code/bin:/cygdrive/c/boot-clj:/cygdrive/c/Users/XXXXX/AppData/Local/GitHubDesktop/bin:/cygdrive/c/apktool:/cygdrive/c/sciter/bin/64:/cygdrive/c/nw/nwjs-v0.25.2-win-x64:/cygdrive/c/spark-2.2.0-bin-hadoop2.7/bin:/cygdrive/c/Program Files (x86)/Microsoft Visual Studio 14.0/VC/bin/amd64:/cygdrive/c/Users/XXXXX/AppData/Roaming/Pub/Cache/bin:/cygdrive/c/Projects/go/bin:/cygdrive/c/lein:/cygdrive/c/Program Files (x86)/Microsoft SDKs/F#/4.1/Framework/v4.0
OS=Windows_NT
OneDrive=C:\Users\XXXXX\OneDrive
PATH=C:\cygwin64\usr\local\bin;C:\cygwin64\bin;C:\cuda\v8.0\bin;C:\cuda\v8.0\libnvvp;C:\ProgramData\Oracle\Java\javapath;C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0;C:\Program Files (x86)\NVIDIA Corporation\PhysX\Common;C:\Program Files\nodejs;C:\Program Files\erl9.0\bin;C:\Users\XXXXX\.mix\escripts;C:\WINDOWS\system32;C:\WINDOWS;C:\WINDOWS\System32\Wbem;C:\WINDOWS\System32\WindowsPowerShell\v1.0;C:\Program Files\Git\cmd;C:\Program Files\Dart\dart-sdk\bin;C:\PostgreSQL\pg96\bin;C:\Program Files (x86)\Windows Kits\8.1\Windows Performance Toolkit;C:\Program Files\CMake\bin;C:\Program Files\Java\jdk1.8.0_144\bin;C:\Program Files (x86)\sbt\bin;C:\Go\bin;C:\Program Files (x86)\Elixir\bin;C:\Users\XXXXX\.mix\escripts;C:\Program Files\Git\mingw64\bin;C:\Program Files\dotnet;C:\Ruby24-x64\bin;C:\Users\XXXXX\AppData\Local\Programs\Python\Python36\Scripts;C:\Users\XXXXX\AppData\Local\Programs\Python\Python36;C:\Users\XXXXX\AppData\Roaming\local\bin;C:\Users\XXXXX\.cargo\bin;C:\Users\XXXXX\scoop\apps\yarn\current\Yarn\bin;C:\Users\XXXXX\scoop\apps\yarn\current\bin;C:\Users\XXXXX\scoop\shims;C:\Users\XXXXX\AppData\Local\Microsoft\WindowsApps;C:\emacs\bin;C:\flutter\bin;C:\Users\XXXXX\.cargo\bin;C:\Users\XXXXX\AppData\Local\atom\bin;C:\Users\XXXXX\AppData\Roaming\npm;C:\Program Files\Microsoft VS Code\bin;C:\boot-clj;C:\Users\XXXXX\AppData\Local\GitHubDesktop\bin;C:\apktool;C:\sciter\bin\64;C:\nw\nwjs-v0.25.2-win-x64;C:\spark-2.2.0-bin-hadoop2.7\bin;C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\bin\amd64;C:\Users\XXXXX\AppData\Roaming\Pub\Cache\bin;C:\Projects\go\bin;C:\lein;C:\Program Files (x86)\Microsoft SDKs\F#\4.1\Framework\v4.0
PATHEXT=.COM;.EXE;.BAT;.CMD;.VBS;.VBE;.JS;.JSE;.WSF;.WSH;.MSC;.RB;.RBW
PKG_CONFIG=C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v8.0\include
PRINTER=HPAF47EF (HP Deskjet 3520 series)
PROCESSOR_ARCHITECTURE=AMD64
PROCESSOR_IDENTIFIER=AMD64 Family 21 Model 2 Stepping 0, AuthenticAMD
PROCESSOR_LEVEL=21
PROCESSOR_REVISION=0200
PROFILEREAD=true
PROGRAMFILES=C:\Program Files
PROMPT=$P$G
PS1=\[\e]0;\w\a\]\n\[\e[32m\]\u@\h \[\e[33m\]\w\[\e[0m\]\n\$
PSModulePath=C:\Program Files\WindowsPowerShell\Modules;C:\WINDOWS\system32\WindowsPowerShell\v1.0\Modules
PUBLIC=C:\Users\Public
PWD=/cygdrive/c/Projects
ProgramData=C:\ProgramData
ProgramFiles(x86)=C:\Program Files (x86)
ProgramW6432=C:\Program Files
SBT_HOME=C:\Program Files (x86)\sbt\
SC2PATH=C:\Program Files (x86)\StarCraft II
SESSIONNAME=Console
SHELL=/bin/bash
SHLVL=1
SPARK_HOME=C:\spark-2.2.0-bin-hadoop2.7\
STACK_ROOT=C:\sr
SYSTEMDRIVE=C:
SYSTEMROOT=C:\WINDOWS
TEMP=C:\cygwin64\tmp
TERM=cygwin
TMP=C:\cygwin64\tmp
TZ=America/New_York
USER=XXXXX
USERDOMAIN=DESKTOP-K8K84AK
USERDOMAIN_ROAMINGPROFILE=DESKTOP-K8K84AK
USERNAME=XXXXX
USERPROFILE=C:\Users\XXXXX
VBOX_MSI_INSTALL_PATH=C:\Program Files\Oracle\VirtualBox\
VS140COMNTOOLS=C:\Program Files (x86)\Microsoft Visual Studio 14.0\Common7\Tools\
WINDIR=C:\WINDOWS
_=/cygdrive/c/Windows/system32/cmd
```
mount 
```
C:/cygwin64/bin on /usr/bin type ntfs (binary,auto)
C:/cygwin64/lib on /usr/lib type ntfs (binary,auto)
C:/cygwin64 on / type ntfs (binary,auto)
C: on /cygdrive/c type ntfs (binary,posix=0,user,noumount,auto)
D: on /cygdrive/d type ntfs (binary,posix=0,user,noumount,auto)
```

cygcheck -c 
```
Cygwin Package Information
Package                                 Version               Status
_autorebase                             001006-1              OK
alternatives                            1.3.30c-10            OK
base-cygwin                             3.8-1                 OK
base-files                              4.2-4                 OK
bash                                    4.4.12-3              OK
bzip2                                   1.0.6-3               OK
ca-certificates                         2.16-1                OK
coreutils                               8.26-2                OK
csih                                    0.9.9-1               OK
curl                                    7.55.1-1              OK
cygrunsrv                               1.62-1                OK
cygutils                                1.4.16-2              OK
cygwin                                  2.9.0-3               OK
dash                                    0.5.9.1-1             OK
diffutils                               3.5-2                 OK
editrights                              1.03-1                OK
file                                    5.30-1                OK
findutils                               4.6.0-1               OK
gawk                                    4.2.0-1               OK
getent                                  2.18.90-4             OK
git                                     2.14.2-2              OK
grep                                    3.0-2                 OK
groff                                   1.22.3-1              OK
gzip                                    1.8-1                 OK
hostname                                3.13-1                OK
info                                    6.5-2                 OK
ipc-utils                               1.0-2                 OK
less                                    487-1                 OK
libargp                                 20110921-3            OK
libattr1                                2.4.46-1              OK
libblkid1                               2.25.2-2              OK
libbz2_1                                1.0.6-3               OK
libcom_err2                             1.42.12-2             OK
libcrypt0                               2.1-1                 OK
libcurl4                                7.55.1-1              OK
libdb5.3                                5.3.28-2              OK
libedit0                                20130712-1            OK
libexpat1                               2.2.3-1               OK
libffi6                                 3.2.1-2               OK
libgc1                                  7.6.0-1               OK
libgcc1                                 6.4.0-1               OK
libgdbm4                                1.12-1                OK
libgmp10                                6.1.2-1               OK
libgssapi_krb5_2                        1.15.1-1              OK
libguile17                              1.8.8-3               OK
libguile2.0_22                          2.0.14-1              OK
libiconv                                1.14-3                OK
libiconv2                               1.14-3                OK
libidn2_0                               2.0.4-1               OK
libintl8                                0.19.8.1-2            OK
libisl13                                0.14.1-1              OK
libk5crypto3                            1.15.1-1              OK
libkrb5_3                               1.15.1-1              OK
libkrb5support0                         1.15.1-1              OK
libltdl7                                2.4.6-5               OK
liblzma5                                5.2.3-1               OK
libmetalink3                            0.1.2-1               OK
libmpc3                                 1.0.3-1               OK
libmpfr4                                3.1.5-1p10            OK
libncursesw10                           6.0-11.20170617       OK
libnghttp2_14                           1.23.1-1              OK
libopenldap2_4_2                        2.4.42-1              OK
libopenssl100                           1.0.2k-1              OK
libp11-kit0                             0.23.5-1              OK
libpcre1                                8.40-3                OK
libpipeline1                            1.4.0-1               OK
libpopt-common                          1.16-2                OK
libpopt0                                1.16-2                OK
libpsl5                                 0.18.0-1              OK
libreadline7                            7.0.3-3               OK
librsync-devel                          1.0.0-1               OK
librsync2                               1.0.0-1               OK
libsasl2_3                              2.1.26-11             OK
libsigsegv2                             2.10-2                OK
libsmartcols1                           2.25.2-2              OK
libssh2_1                               1.7.0-1               OK
libssp0                                 6.4.0-1               OK
libstdc++6                              6.4.0-1               OK
libtasn1_6                              4.12-1                OK
libunistring2                           0.9.6-1               OK
libuuid1                                2.25.2-2              OK
login                                   1.11-1                OK
m4                                      1.4.18-1              OK
make                                    4.2.1-2               OK
man-db                                  2.7.5-2               OK
mingw64-x86_64-binutils                 2.29.1.787c9873-1     OK
mingw64-x86_64-gcc-core                 6.4.0-2               OK
mingw64-x86_64-headers                  5.0.2-1               OK
mingw64-x86_64-runtime                  5.0.2-1               OK
mingw64-x86_64-windows-default-manifest 6.4-1                 OK
mingw64-x86_64-winpthreads              5.0.2-1               OK
mintty                                  2.8.0-0               OK
ncurses                                 6.0-11.20170617       OK
openssh                                 7.6p1-1               OK
openssl                                 1.0.2k-1              OK
p11-kit                                 0.23.5-1              OK
p11-kit-trust                           0.23.5-1              OK
patch                                   2.7.4-1               OK
perl                                    5.26.1-1              OK
perl-Carp                               1.38-2                OK
perl-Error                              0.17025-2             OK
perl-TermReadKey                        2.37-2                OK
perl_autorebase                         5.26.1-1              OK
perl_base                               5.26.1-1              OK
publicsuffix-list-dafsa                 20170828-1            OK
rebase                                  4.4.2-1               OK
rsync                                   3.1.2-1               OK
run                                     1.3.4-2               OK
sed                                     4.4-1                 OK
tar                                     1.29-1                OK
terminfo                                6.0-11.20170617       OK
tzcode                                  2016j-1               OK
tzdata                                  2017b-1               OK
unzip                                   6.0-17                OK
util-linux                              2.25.2-2              OK
vim-minimal                             8.0.1157-1            OK
which                                   2.20-2                OK
xz                                      5.2.3-1               OK
zlib0                                   1.2.11-1              OK
```

**Comment by fdopen** [comment]
It's probably because your folder (`C:\Projects`) has the "wrong" user rights (cygwin posix emulation doesn't interact very well with native windows access rights).

If you encounter further problems, rename it and create a new folder with the same name (with cmd.exe  shell):
```batch
set HOME=
C:\cygwin64\bin\bash.exe -l
exit # back to cmd.exe
cd C:\
move "cygwin64\home\YourUsername" Projects
exit
```
Then start a cygwin shell and try `opam init ...` again. If everything works this time, you can move the content of your old folder to the new folder.

**Comment by tensor-programming** [comment]
No it has nothing to do with user writes on the project folder.  I have encountered write protection problems before with this setup for other tools and services on windows and have since changed the access rights on the projects folder so that this will never be a problem again.  Since I work with so many different programming languages, (many that do not work well on windows), I try to keep all of my projects in one area and of course, that was one of the first things I had to address. 

Anyways, thank you for your help.
